worker/lib/workers/qemu.ts

```diff
@@ -229,22 +238,24 @@ class QemuWorker extends EventEmitter implements Leviathan.Worker {
 	}
 
 	private async createBridge(bridgeName: string, bridgeAddress: string): Promise<void> {
-		return new Promise((resolve, reject) => {
+		return new Promise<void>((resolve, reject) => {
 			spawn(
 				'brctl', ['addbr', bridgeName]
 			).on('exit', (code) => {
 				if (code == 0) {
+					console.log(`added bridge ${bridgeName}...`);
```

Comment:
This is more of a debug statement, we don't need to know when it succeeds, only when it fails.

Reply:
Debug like `console.debug` or like `if (this.qemuOptions.debug)` or like remove it?

I'm fine with any option, including removing it. I just wanted to understand your suggestion.

Reply:
@klutchell I would say either removing them, or changing them to debug level using `console.debug()` (and this applies for the debug print statements I accidentally left in as well).